stations.csv

```diff
@@ -5075,7 +5075,7 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 6675;Pamplona/Iruña;pamplona-iruna;7180100;;42.825122;-1.661889;;ES;Europe/Madrid;f;f;f;t;f;f;ESPNA;;f;;f;7100080;f;ezwgdd;t;ESPNA;t;;f;;f;;f;;f;;f;;f;;;;f;;f;80100;t;;f;ESPNA;f;;f;f;;;;;Pamplune;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv6675;
 6676;Port-Bou;port-bou;;;;;;ES;Europe/Madrid;f;f;f;f;f;f;ESPOB;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;4986;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv4986;
 6677;Tarragona Estació;tarragona-estacio;7171500;;41.111212;1.253527;24444;ES;Europe/Madrid;f;t;f;t;f;t;ESQGN;;f;;f;7100062;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;71500;t;;f;ESQGN;f;;f;f;;;;;Tarragone;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv6677;
-6678;Ribas de Freser;ribas-de-freser;7177303;;;;;ES;Europe/Madrid;f;f;f;f;f;f;ESRDF;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;ESRIB;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv6678;
+6678;Ribas de Freser;ribas-de-freser;7177303;;42.301158;2.166078;;ES;Europe/Madrid;f;f;f;f;f;f;ESRDF;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;ESRIB;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv6678;
```

Comment:
The official catalan name for this one is Ribes de Freser

Reply:
This should be fixed as well